### PR TITLE
update the container documentation

### PR DESCRIPTION
**Pull request by yveskerwyn**

### CODE REVIEW

File: docs/interacting/commands/container.md [hunk 1]
```diff
@@ -74,22 +78,22 @@ Values:
...
-  - If none it will automatically be set to core-x, x being the ID of the container
+- **{hostname}**: Specific hostname you want to give to the container
+  - If none it will automatically be set to `core-x`, x being the ID of the container
+
+- **{privileged}**: True/False. When True the container has privileged access to the host, the default is False, isolating the container from the host.
 
-- **storage**: URL to the ARDB storage cluster to mount, e.g. `ardb://hub.gig.tech:16379`
+- **{storage}**: URL to the ARDB storage cluster to mount, e.g. `ardb://hub.gig.tech:16379`
   - If not provided the default one from the Zero-OS main configuration will be used, see the documentation about `storage` in [Main Configuration](../../config/main.md) for more details
+- **{tags}**: List of key-values that you can attach to a container, can be used to to search all containers matching a specified set of tags; see the `find()` command
```
Comment (muhamadazmy) [inline docs/interacting/commands/container.md]: Not key values, they are just labels (strings)

File: docs/interacting/commands/container.md [hunk 2]
```diff
@@ -74,22 +78,22 @@ Values:
 
   Example: `port=[8080: 80, 7000:7000]`
 
-- **hostname**: Specific hostname you want to give to the container
-  - If none it will automatically be set to core-x, x being the ID of the container
+- **{hostname}**: Specific hostname you want to give to the container
+  - If none it will automatically be set to `core-x`, x being the ID of the container
+
+- **{privileged}**: True/False. When True the container has privileged access to the host, the default is False, isolating the container from the host.
```
Comment (muhamadazmy) [inline docs/interacting/commands/container.md]: I would say 'privileged access to the host *devices*'

Comment (yveskerwyn) [inline docs/interacting/commands/container.md]: OK updated

File: docs/interacting/commands/container.md (hunk 1 above)
Comment (yveskerwyn) [inline docs/interacting/commands/container.md]: OK updated

**Pull request merged**